newIDE/app/src/Profile/EditProfileDialog.js

```diff
@@ -39,6 +40,11 @@ export const getUsernameErrorText = (error: ?AuthError) => {
   return undefined;
 };
 
+const simpleUrlRegex = /^https?:\/\/[^ ]+$/;
```

Comment:
Should we enforce the https? Feels more secure and not sure it restrains the possibilities

Reply:
It's a very very good idea to enforce https 👍

Reply:
yep